browser/extensions/api/brave_shields_api.cc

```diff
@@ -66,31 +66,31 @@ BraveShieldsUrlCosmeticResourcesFunction::Run() {
 
 std::unique_ptr<base::ListValue> BraveShieldsUrlCosmeticResourcesFunction::
     GetUrlCosmeticResourcesOnTaskRunner(const std::string& url) {
```

Comment:
Should be able to remove this (and `BraveShieldsHiddenClassIdSelectorsFunction` below) altogether, since they won't be needed in JS anymore

Reply:
yes, I will do, just wanted to make sure we are fine with removing extensions methods

Reply:
removed